static/js/lib/search.js

```diff
@@ -173,7 +173,8 @@ const LIST_QUERY_FIELDS = [
   "topics"
 ]
 
-const SUGGEST_FIELDS = ["title", "short_description"]
+const LEARN_SUGGEST_FIELDS = ["title.trigram", "short_description.trigram"]
+const CHANNEL_SUGGEST_FIELDS = ["suggest_field1", "suggest_field2"]
```

Comment:
What do you think of adding `suggest_field_1` and `suggest_field_2` aliases to the `LEARNING_RESOURCE_TYPE` schema for consistency, and then you wouldn't need separate suggest field arrays for learning and channels?

Reply:
I think that will cause data from learning resource fields to be used to generate suggestions for course_catalog fields and vice versa.